test/unit/spec/EnrollChoices_spec.js

```diff
@@ -280,6 +285,11 @@ function (Okta, OktaAuth, Util, EnrollChoicesForm, Beacon, Expect, Router,
               .toHaveBeenCalledWith('signin/enroll/google/token%3Asoftware%3Atotp', { trigger: true });
           });
         });
+        itp('has link to skip set up during grace period when some are required', function () {
```

Comment:
and this could be `has skip setup link when it is present in the response and 1 required factor is enrolled and some required are not enrolled yet`